package.json

```
@@ -36,9 +34,6 @@
         "cordova-android": ">=3.6.0",
         "cordova-windows": ">=4.4.0"
       },
-      "<6.0.0": {
-        "cordova-ios": "<6.0.0"
-      },
       "6.0.0": {
```

this should be bumped to 7 or it won't install
```suggestion
      "7.0.0": {
```

---

I will do that change in a separate PR.